client/src/backend/containers/project/ProjectExportations.js

```diff
@@ -0,0 +1,167 @@
+        requests.beProjectExportations,
+        state.entityStore
+      ),
+      projectExportationsMeta: meta(
+        requests.beProjectExportations,
+        state.entityStore
+      ),
+      exportTargets: select(requests.beExportTargets, state.entityStore)
+    };
+  };
+
+  static displayName = "Project.Exportations.List";
+
+  static propTypes = {
+    // the following arguments are required to render the component,
+    // but can be null until the async network request is fulfilled
+    /* eslint-disable react/require-default-props */
+    projectExportations: PropTypes.array,
+    projectExportationsMeta: PropTypes.object,
+    exportTargets: PropTypes.array,
+    /* eslint-enable react/require-default-props */
+    project: PropTypes.object.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    projectExportationsPerPage: PropTypes.number
+  };
+
+  static defaultProps = {
+    projectExportationsPerPage: 20
+  };
+
+  componentDidMount() {
+    this.fetchExportations(1);
+    this.fetchExportTargets();
+  }
+
+  get exportTargetSelectOptions() {
+    const targets = [{ label: "", value: "", internalValue: "" }];
+    const { exportTargets } = this.props;
+
+    if (!isArray(exportTargets)) return targets;
+
+    exportTargets.forEach(exportTarget =>
+      targets.push({
+        label: exportTarget.attributes.name,
+        value: exportTarget.id,
+        internalValue: exportTarget.id
+      })
+    );
+    return targets;
+  }
+
+  pageChangeHandlerCreator = page => {
+    return () => this.fetchExportations(page);
+  };
+
+  dispatch(action) {
+    this.props.dispatch(action);
+  }
+
+  fetchExportTargets() {
+    const action = request(exportTargetsAPI.index(), requests.beExportTargets);
+    this.dispatch(action);
+  }
+
+  fetchExportations(page) {
+    const pagination = {
+      number: page,
+      size: this.props.projectExportationsPerPage
+    };
+    const action = request(
+      projectsAPI.project_exportations(this.props.project.id, {}, pagination),
+      requests.beProjectExportations
+    );
+    this.dispatch(action);
+  }
+
+  render() {
+    const active = false;
+    const {
+      projectExportations,
+      project,
+      projectExportationsMeta
+    } = this.props;
+
+    if (!projectExportations || !projectExportationsMeta) return null;
+
+    const { pagination } = projectExportationsMeta;
+
+    return (
+      <Authorize
+        entity={project}
+        ability="manageProjectExportations"
+        failureNotification
+        failureRedirect={lh.link("backendProject", project.id)}
+      >
+        <FormContainer.Form
+          className="form-secondary"
+          name={requests.beProjectExportationCreate}
+          model={{
+            attributes: {
+              project_id: project.id,
+              export_target_id: ""
+            }
+          }}
+          update={() => null}
+          create={projectExportationsAPI.create}
+          onSuccess={() => this.fetchExportations(1)}
```

Comment:
Is it ok that on successful creation of a project exportation, the form fetches the first page so the new project exportation can be seen in the project exportations list immediately after?

Reply:
Yes, this is OK.